plugins/source/aws/resources/services/applicationautoscaling/scalable_targets.go

```diff
@@ -0,0 +1,51 @@
+package applicationautoscaling
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/applicationautoscaling"
+	"github.com/aws/aws-sdk-go-v2/service/applicationautoscaling/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func ScalableTargets() *schema.Table {
+	tableName := "aws_applicationautoscaling_scalable_targets"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/autoscaling/application/APIReference/API_ScalableTarget.html`,
+		Resolver:    fetchScalableTargets,
+		Multiplex:   client.ServiceAccountRegionNamespaceMultiplexer(tableName, "application-autoscaling"),
+		Transform:   transformers.TransformWithStruct(&types.ScalableTarget{}, transformers.WithPrimaryKeys("ResourceId")),
```

Comment:
`ResourceId` can be bunch of things, some ARN, some not. Refer to the URL in the description for details.